src/main/zapHomeFiles/hud/tools/history.js

```diff
@@ -6,14 +6,13 @@
 
 const History = (function () {
 	// Constants
-	// todo: could probably switch this to a config file?
```

Comment:
This and the following can be removed from all tools.

Reply:
Ok I will